### PR TITLE
Productionize Homepage Prototype Styles

### LINKED ISSUE #1708: Convert homepage prototype CSS to patterns and utility classes
There is a small amount of [custom CSS in the homepage prototype](https://github.com/cloudfour/cloudfour.com-patterns/blob/v-next/src/prototypes/home-page/example/example.scss) that needs to be converted to production code for patterns or utility classes before the wordpress work can continue.

### PR DESCRIPTION
**Pull request by spaceninja**
## Overview

This PR takes the custom CSS for the homepage content prototype and
converts it to use a combination of new component modifiers and utility
classes, so the wordpress template can be built.

## Testing

Compare the [homepage content prototype](https://deploy-preview-1710--cloudfour-patterns.netlify.app/iframe.html?id=prototypes-home-page-content--example&args=&viewMode=story) on the preview deploy to the [current version](https://v-next--cloudfour-patterns.netlify.app/iframe.html?id=prototypes-home-page-content--example&args=&viewMode=story). There should be no visual changes. Check in the following browsers:

- [ ] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Desktop Safari
- [ ] Mobile Safari

---

- Fixes #1708

### CODE REVIEW

File: src/objects/media/media.scss [hunk 1]
```diff
@@ -36,6 +36,16 @@
...
 }
 
+/**
+ * Modifier: Jaunty
+ *
+ * Applies a fun rotation to the media object
+ */
+
+.o-media--jaunty .o-media__object {
+  transform: rotate(-5deg);
```
Comment (tylersticka) [inline src/objects/media/media.scss:46]: Should we apply the inverse when the media object is reversed?

File: src/objects/media/media.stories.mdx [hunk 2]
```diff
@@ -139,3 +141,20 @@ In this example, the object is [a Calendar Date component](/docs/components-cale
...
   </Story>
 </Canvas>
+
+## Jaunty
+
+By adding the `o-media--jaunty` class, the media object will have a fun little rotation applied to it.
+In this example, we're using it for a Promo.
+
+You may want to add some of our [spacing utility classes](/docs/utilities-spacing--margin) to the media object to compensate for the rotation.
+In this example, we've used `u-space-inline-end-1` and `u-space-inline-start-n4` to add a bit more space between the image and the text, and keep the image from hitting the left side of the viewport on small screens.
```
Comment (tylersticka) [inline src/objects/media/media.stories.mdx:151]: It feels strange to me to create a modifier but then say you probably need utility classes. Any reason not to make adjustments using the modifier class?

Comment (tylersticka) [reply src/objects/media/media.stories.mdx:151]: That sounds a bit nicer to me.

File: src/objects/media/media.stories.mdx [hunk 3]
```diff
@@ -21,19 +21,19 @@ const exampleText =
 // respond to args more dynamically than other demos.
 const imageDemoTransformSource = (_src, storyContext) => {
   const { args } = storyContext;
-  const withStr = args.reverse
+  const withStr = args?.reverse
```
Comment (spaceninja) [inline src/objects/media/media.stories.mdx:24]: This was causing the Docs page for the Media object to fail to render.

File: src/objects/media/media.scss (hunk 1 above)
Comment (spaceninja) [reply src/objects/media/media.scss:59]: Done!

File: src/objects/media/media.stories.mdx (hunk 2 above)
Comment (spaceninja) [reply src/objects/media/media.stories.mdx]: Done!

File: src/objects/media/media.stories.mdx (hunk 3 above)
Comment (gerardo-rodriguez) [reply src/objects/media/media.stories.mdx:24]: Thank you, this seems to be happening on multiple components and objects, I’ve noticed.

**Review by tylersticka — APPROVED** [review]
LGTM! Also forgot to mention that "jaunty" is super fun 😄

**Pull request merged**